mautic/mautic
# Fix elfinder folder perms

| Q                                      | A
| -------------------------------------- | ---
| Bug fix?                               | no
| New feature?                           | yes
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | yes
| Related user documentation PR URL      | N.A.
| Related developer documentation PR URL | N.A.
| Issue(s) addressed                     | N.A.




#### Description:
Need to set appropriate permissions for newly created folders using el-finder.
The permissions should be:
Folders: 777
Files: 666



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Click on browse to open el finder (file explorer) in new browser window
3. Create a new folder and upload some image files in it
4. Check folder and file permissions




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10410"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
